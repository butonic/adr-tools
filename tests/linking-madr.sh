adr init docs madr
adr new First Record
adr new Second Record
adr new Third Record
adr link 4 Amends 2 "Amended by"
adr link 4 Clarifies 3 "Clarified by"
tail -5 docs/0002-first-record.md
tail -5 docs/0003-second-record.md
tail -6 docs/0004-third-record.md
