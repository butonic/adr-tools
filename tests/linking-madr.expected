adr init docs madr
docs/0001-record-architecture-decisions.md
adr new First Record
docs/0002-first-record.md
adr new Second Record
docs/0003-second-record.md
adr new Third Record
docs/0004-third-record.md
adr link 4 Amends 2 "Amended by"
adr link 4 Clarifies 3 "Clarified by"
tail -5 docs/0002-first-record.md
## Links <!-- optional -->

* [Link type] [Link to ADR] <!-- example: Refined by [ADR-0005](0005-example.md) -->
* … <!-- numbers of links can vary -->
* Amended by [Third Record](0004-third-record.md)
tail -5 docs/0003-second-record.md
## Links <!-- optional -->

* [Link type] [Link to ADR] <!-- example: Refined by [ADR-0005](0005-example.md) -->
* … <!-- numbers of links can vary -->
* Clarified by [Third Record](0004-third-record.md)
tail -6 docs/0004-third-record.md
## Links <!-- optional -->

* [Link type] [Link to ADR] <!-- example: Refined by [ADR-0005](0005-example.md) -->
* … <!-- numbers of links can vary -->
* Amends [First Record](0002-first-record.md)
* Clarifies [Second Record](0003-second-record.md)
